### PR TITLE
fix(tree-table): added actions support

### LINKED ISSUE #4075: TreeTable actions disappear in responsive view
Actions are available in normal table view:
image

But when the responsive view is used, only the "Show row details" button is available:
image

Here is an example on codesandbox.io:
https://codesandbox.io/s/floral-shadow-fvii3?file=/index.js

Is this the intended behaviour?

### PR DESCRIPTION
**Pull request by mattnolting**
closes #4075 

Previews start here: https://patternfly-pr-4135.surge.sh/components/table#tree-table

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]
this is magical

**Review by mcoker — COMMENTED** [review]
Looks really great!!

Left a couple of questions about vars.

**Review by mcarrano — CHANGES_REQUESTED** [review]
I think this works @mattnolting , but I was a little uncomfortable with the two kabob icons (horz and vertical) grouped like that. I brainstormed with @mceledonia a bit and we came up with another idea. Can we take the "show more (...)" kabob and move it so that it's left aligned after the node label? The result would look something like this...

Tree-table-actions

We recommed using a 24px spacer between the text and the (...) icon button. What do you think? Note: I'd suggest this a consistent placement whether there are actions on the row or not.

File: src/patternfly/components/Table/table-tree-view.scss [hunk 1]
```diff
@@ -72,7 +72,8 @@ $pf-c-tree-view--MaxDepth: 10;
   margin-right: var(--pf-c-table__tree-view-icon--MarginRight);
 }
 
-.pf-c-table__tree-view-details-toggle {
+.pf-c-table__tree-view-details-toggle,
+.pf-c-table__actions {
```
[the remark this answers is not in the document]
Comment (mattnolting) [reply src/patternfly/components/Table/table-tree-view.scss:76]: Screen Shot 2021-06-09 at 3 14 19 PM

[the remark this answers is not in the document]
Comment (mcoker) [reply src/patternfly/components/Table/table-tree-view.scss:76]: the element in the css is `.pf-c-table__actions` (plural)

[the remark this answers is not in the document]
Comment (mattnolting) [reply src/patternfly/components/Table/table-tree-view.scss]: Ah, missed it

[the remark this answers is not in the document]
Comment (mattnolting) [reply src/patternfly/components/Table/table-tree-view.scss]: Updated

**Review by nicolethoen — APPROVED** [review]

**Review by mcarrano — APPROVED** [review]
I like this much better. Thanks for making that change @mattnolting .  

@mceledonia can you also take a look?

File: src/patternfly/components/Table/table-tree-view.scss [hunk 2]
```diff
@@ -157,13 +169,18 @@ $pf-c-tree-view--MaxDepth: 10;
 
   tr:not([hidden]) {
     display: grid;
+    grid-template-columns: 1fr max-content;
   }
 
   // target node rows
   tr[aria-expanded] {
     .pf-c-table__tree-view-title-cell {
       --pf-c-table--cell--PaddingTop: var(--pf-c-table--m-tree-view-grid--tbody--node--PaddingTop);
```
Comment (mcoker) [inline src/patternfly/components/Table/table-tree-view.scss:178]: Should we update this var to match the one below? Also we could nest the rule below in this block.

**Comment by mcoker** [comment]
Looks like the thead row border doesn't extend to the end in safari. 

Screen Shot 2021-06-09 at 4 20 24 PM

**Comment by mattnolting** [comment]
@mcoker updated

**Review by mcoker — APPROVED** [review]
L🍕TM!!

**Pull request merged**